loader/src/hooks/MessageBoxFix.cpp

```diff
@@ -28,11 +28,25 @@ static void __cdecl fixedErrorHandler(int code, char const* description) {
     );
 }
 
-$execute {
-    // TODO: i'm pretty sure this patch only works on 2.113?
-    //(void)Mod::get()->patch(
-    //    reinterpret_cast<void*>(geode::base::getCocos() + 0x19feec), toByteArray(&fixedErrorHandler)
-    //);
+$execute {
+    // updated for 2.204
+    // check xrefs to "GLFWError #%d Happen, %s\n", now there's two functions with the same exact behaviour, one is a member function though...
+	// call ds:MessageBoxW 
+    // patch MessageBoxW to MessageBoxA
+    // geode::base::getCocos() + 0x122600 = MessageBoxA in .idata
+    // geode::base::getCocos() + 0x1225DC = MessageBoxW in .idata
+    const uint32_t importedMessageBoxA = (geode::base::getCocos() + 0x122600);
```

Comment:
why the ()

Reply:
forgot to remove because i casted before, fixing rn